migration/idempotent/001-base.sql

```diff
@@ -1036,7 +1077,9 @@ GRANT EXECUTE ON FUNCTION prom_api.jsonb(prom_api.label_array) TO prom_reader;
 
 --Returns the label_array given a series_id
 CREATE OR REPLACE FUNCTION prom_api.labels(series_id BIGINT)
-RETURNS prom_api.label_array AS $$
+    RETURNS prom_api.label_array
+    SET search_path = pg_catalog
```

Comment:
I looked into this doc on inlining when I was trying to figure out what could/should be inlined: https://wiki.postgresql.org/wiki/Inlining_of_SQL_functions.

According to that, for functions used in the scalar position (e.g. `SELECT labels(some_id);`), inlining will not occur if the body contains a `FROM`:
> the body contains no aggregate or window function calls, no subqueries, no CTEs, no FROM clause or references to any table or table-like object, none of GROUP BY, HAVING, ORDER BY, DISTINCT, LIMIT, OFFSET, UNION, INTERSECT, EXCEPT

It seems as though for functions used in the table position (e.g. `SELECT * FROM labels(some_id);`), that restriction is lifted, but there is the following requirement:
> the function is declared RETURNS SETOF or RETURNS TABLE

Honestly I don't know enough to be able to say definitively, but it doesn't seem like this can be inlined.